src/js/text/main.html

```diff
@@ -33,6 +33,11 @@
                 </div>
                 <div class="docs--meta docs--standfirst-read-more">Full description and credits</div>
             </div>
+            <div class='docs--actions'>
+              <div class='docs--actions__trailer docs__shows-trailer'>
+                <span class='docs--actions__trailer__duration'>1:08</span>
```

Comment:
Yep that'd make sense — @gids is incorporating the actual youtube video in a later PR